localenv/cloud-nine-wallet/seed.yml

```diff
@@ -33,6 +36,16 @@ accounts:
     path: accounts/wbdc
     postmanEnvVar: wbdcPaymentPointer
     assetCode: USD
-fees:
-  - fixed: 1
-    percentage: 0.02
+rates:
+  EUR:
+    MXN: 18.68
+    USD: 1.10
+  USD:
+    MXN: 18.68
+    EUR: 0.91
+  MXN:
+    USD: 0.059
+    EUR: 0.054
+sendingFee:
+    fixed: 0
+    percentage: 0
```

Comment:
IMO I think we should keep this at 0 until we start properly showing the mock account balances